### DISCUSSION TITLE
Question on the camera parameters

### DISCUSSION THREAD

**Discussion by shota0610** [HumanSupportRobot/QandA#92] [Q&A]
I'm using the HSR simulator with ROS noetic. I would be grateful if you could provide information of the parallax and focal length of the stereo camera installed in HSR. I checked the HSR urdf file, but I couldn't find it. Your help would be greatly appreciated.

Thank you.
Shota

**Comment by yutaro-ishida** [comment]
`head_l_stereo_camera_frame` and `head_r_stereo_camera_frame` are the optical coordinate system of the stereo cameras.
Please refers to the distance between these two frames for the parallax (baseline).
In the case of HSR Gazebo simulator, `head_X_stereo_camera_frame` is identical to `head_X_stereo_camera_link` and the distance between two frames is 0.14m.
https://github.com/hsr-project/hsrb_gazebo_bringup/blob/master/launch/include/hsrb_bringup.xml#L71-L74
https://github.com/hsr-project/hsrb_description/blob/master/urdf/head_v2/head.urdf.xacro#L114-L120

Focal length is provided by Gazebo's camera plugin. Please refer `CameraInfo` topic of each camera.
